bundles/org.openhab.binding.pushover/README.md

```diff
@@ -74,6 +75,7 @@ For priority `2` only, the action returns a `String` value (the `receipt`) if th
 For other priorities, the action always returns an empty `String`.
 
 - `cancelPriorityMessage(String receipt)` - This method is used to cancel an emergency priority message.
+
```

Comment:
Why this new line? The sentence below is directly related to `cancelPriorityMessage`. I think the line needs to be moved up instead, since I can see it's already rendered wrongly: https://next.openhab.org/addons/bindings/pushover/#thing-actions

Reply:
Mistakingly interpeted this as a seperate sentence and a new line is needed after a list. Reverted.

Reply:
👍 Will create another PR for fixing that existing README issue.

Reply:
Oh, i can add it here. if you wait 5 mins for merging